aioconsole/events.py

```diff
@@ -78,8 +80,26 @@ def set_interactive_policy(*, locals=None, banner=None, serve=None,
         prompt_control=prompt_control)
     asyncio.set_event_loop_policy(policy)
 
+def exec_pythonstartup(locals):
+    filename = os.environ.get('PYTHONSTARTUP')
+    if filename:
+        if os.path.isfile(filename):
+            with open(filename) as fobj:
+                startup_file = fobj.read()
+            try:
+                exec(startup_file, globals(), locals)
```

Comment:
Maybe `locals_dict` so it is not mistaken for the `locals` builtin function?

Reply:
That is a easy change to make; I was being consistent with existing code where ``locals`` was being passed around, already smashing the built-in function.

Reply:
Yea I noticed that :sweat_smile: but here `exec(startup_file, globals(), locals)` really looks like a typo